task1.js

```diff
@@ -0,0 +1,119 @@
+const express = require('express');
+const bp = require('body-parser');
+const Joi = require('joi');
+const app=express();
+const port=3000;
+
+app.use(bp.json());
+
+const user1={
```

Comment:
Try to avoid making your code be responsible for all in one file.
Keep application data separately from code.
In such a way, it is much easier to understand your code. 

For this purpose, manage users via repository:
1. create dir "resources/users", create file users.csv 
2. move users data to resources/users/users.csv
3. create dir repository, create file repository/users-store.js
4. create a function that will load users from users.csv on server.js start and store them back when it updates. (Use Streams for working with the file)